vmc.sh

```diff
@@ -17,6 +19,7 @@ cmake "$SOURCEDIR"                             \
     ${CXXSTD:+-DCMAKE_CXX_STANDARD=$CXXSTD}
 
 cmake --build . -- ${JOBS:+-j$JOBS} install
+ln -s $INSTALLROOT/lib/libVMCLibrary.so $INSTALLROOT/lib/libVMC.so
```

Comment:
Why is this link needed?

Reply:
I see, that you commented this above.

Reply:
It's when ROOT macros from AliRoot or AliPhysics do something like `gSystem->Load("libVMC")`